content/fr/abstraction.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Abstraction
+status: Feedback Appreciated
+category: Property
+tags: ["fundamental", "", ""]
+---
+
+Dans le contexte informatique, une abstraction est une représentation qui masque les détails spécifiques à un consommateur de services (un consommateur étant un programme informatique ou un humain), rendant un système plus générique, et ainsi facilement compréhensible.
+Le système d'exploitation de votre ordinateur portable en est un bon exemple.
+Il masque tous les détails du fonctionnement de votre ordinateur.
+
+Vous n'avez pas besoin d'avoir une compréhension approfondie du processeur, de la mémoire et du fonctionnement des programmes, vous utilisez simplement le système d'exploitation et le système d'exploitation s'occupe des détails.
+Tous ces détails sont cachés sous le « capot » ou l'abstraction du système d'exploitation.
+
+Généralement, les systèmes ont plusieurs couches d'abstraction.
+Cela simplifie considérablement le développement.
+Lors de la programmation, les développeurs construisent des composants compatibles avec une couche d'abstraction particulière et n'ont pas à se soucier de tous les détails sous-jacents qui peuvent être très hétérogènes.
+Si cela fonctionne avec la couche d'abstraction, cela fonctionne avec le système — peu importe ce qui se trouve sous le capot.
```

Comment:
to avoid repetition with L13, could we change it for "peu importe ce qui se trouve en dessous."?

Reply:
Pour remplacer `capot`, j'hésite entre `dessous` ou `derrière`.

Reply:
Mise à jour effectuée